content/en/docs/concepts/overview/what-is-kubernetes.md

```diff
@@ -59,16 +59,18 @@ That's how Kubernetes comes to the rescue! Kubernetes provides you with a framew
 
 Kubernetes provides you with:
 
+* **Automated rollouts and rollbacks**  
+You can describe the desired state for your deployed containers using Kubernetes, and it can change the actual state to the desired state at a controlled rate. For example, you can automate Kubernetes to create new containers for your deployment, remove existing containers and adopt all their resources to the new container.
+* **Workload isolation**  
+Your application runs inside a _pod_, the basic Kubernetes unit of deployment. Kubernetes containers in a Pod can interact with each over locally and without restrictions. You can set up controls to restrict whether and how different Pods are able to interact with each other.
+* **Self-healing**  
+Kubernetes restarts your application if it fails. You can set up your own health checks so that a pod only receives traffic when it is passing those checks, and so that Kubernetes knows when to treat it as broken.
 * **Service discovery and load balancing**  
-Kubernetes can expose a container using the DNS name or using their own IP address. If traffic to a container is high, Kubernetes is able to load balance and distribute the network traffic so that the deployment is stable.
+Kubernetes can expose a service using a DNS name or by giving it its own IP address. As traffic increases, Kubernetes is able to load balance and distribute the network traffic so that the deployment is stable.
```

Comment:
substitute `it` for something like:
by providing a service an IP address.

As traffic increases, Kubernetes balances and distributes ...